engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -67,6 +68,23 @@ public void setReason(String reason)
     this.reason = reason;
   }
 
+  public void setLogLevel(LogLevel logLevel)
```

Comment:
What is the reason to allow changing LogLevel (provide the setter method)? Under what condition Apex needs to change event log level after event was generated?